server/src/main/resources/rules/rules.js

```diff
@@ -506,20 +506,25 @@ function get_pool_priority(pool, consumer) {
         var null_rule_score = 0;
         var mismatch_rule_score = 0;
 
-        log.debug("Evaluating attribute {} with weight {}", attr, attrs[attr]);
-        if (unsatisfiedSet.length === 0 && poolSet.length === 0) {
-            null_rule_score = 0.1;
-            log.debug("NULL rule score for attribute {} and pool {}, which is: {}", [attr, pool.id, null_rule_score]);
+        if(Utils.equalsIgnoreCase(attr, 'support_level') && Utils.equalsIgnoreCase(poolSet[0], 'Layered')) {
+            log.debug("Ignoring the pool prioritization when SLA is Layered", attr, poolSet[0]);
```

Comment:
Similarly to the check for the syspurpose compliance, we shouldn't be checking by hardcoding `Layered` here, but instead checking if pool we are evaluating has the `support_level_exempt` attribute set to true. Something like `pool.retrievePoolAttributeValues("support_level_exempt")` should work here (although I have not tested it, so it needs verification).

Reply:
UPDATE: based on the latest discussion, we should actually remove these rules.js changes all together

Reply:
Removed the changes related to rules.js.